app/controllers/concerns/laying_date_helper.rb

```diff
@@ -1,11 +1,15 @@
 module LayingDateHelper
-  # Gets the earliest laying date for a work package.
+  # Gets the earliest laying date for a thing connected to a laying.
   #
-  # @param [Grom::Node] work_package Grom::Node of type WorkPackage
+  # @param [Grom::Node] laying_connected_thing Grom::Node of type WorkPackage
```

Comment:
Just take out type WorkPackage (as it could be something else).

Reply:
🎉